pkg/clustermanager/cluster_manager.go

```diff
@@ -719,12 +719,17 @@ func (c *ClusterManager) EKSAClusterSpecChanged(ctx context.Context, clus *types
 }
 
 func compareEKSAClusterSpec(ctx context.Context, currentClusterSpec, newClusterSpec *cluster.Spec) (bool, error) {
-	currentVersionsBundle := currentClusterSpec.ControlPlaneVersionsBundle()
-	newVersionsBundle := newClusterSpec.ControlPlaneVersionsBundle()
+	for version, newVersionsBundle := range newClusterSpec.VersionsBundles {
```

Comment:
Can we use here the `cluster.KubernetesVersions`?
At some point I would like for VersionsBundles to contain all versions bundles and not just the ones used by the cluster